plugins/API/API.php

```diff
@@ -89,6 +89,23 @@ public function getMatomoVersion()
         return Version::VERSION;
     }
 
+    /**
+     * Get PHP version
+     * @return array
+     */
+    public function getPhpVersion()
+    {
+        Piwik::checkUserHasSomeViewAccess();
```

Comment:
This would need to be `Piwik::checkUserHasSuperUserAccess();`.

Reply:
Can you elaborate why? I mean, why do I need super user access to get the php version but not to get the matomo version?

Reply:
We're currently only exposing this information to super users. Partially for security reasons. If you needed it for View users, you could develop a simple plugin that defines this API using eg 

```
./console generate:plugin --name="MyPlugin"
./console generate:api
```

You could even put the plugin on the Marketplace https://developer.matomo.org/guides/distributing-your-plugin

Reply:
I agree with @tsteur.
There are many ways a user with view access could find out the Matomo version without the api, but there should be non to find out the php version.